local/kaltura/service.php

```diff
@@ -98,6 +99,7 @@
     'title' => $title,
     'metadata' => $metadata,
     'editor' => $editor,
+    'previewltilauncher' => $previewltilaunchurl->out(),
```

Comment:
oh... we responded to each other on different subjects.
I thought that your hate was towards the `moodle_url` thing. :)